src/labels/CustomLabels.labels

```diff
@@ -1367,6 +1367,14 @@
         <shortDescription>stgRunCleanUpEnableFirstTime</shortDescription>
         <value>Run this data cleanup process after you&apos;ve enabled the Preferred Email requirement for the first time or after a period of not enforcing the requirement.</value>
     </labels>
+    <labels>
+        <fullName>stgSaveSettings</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>stgstgSaveSettings</shortDescription>
```

Comment:
@lifewithryan Extra "stg"?

Reply:
wow, yea...I suppose it didn't balk because it's a description.

Reply:
Fixed